src/index.html

```diff
@@ -6,7 +6,21 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
   <body>
-    
-    
+    <main>
+      <h1>Leaderboard</h1>
+      <section>
+        <h2>Recent scores</h2>
+        <button id="refresh">Refresh</button>
+        <div class="scores">
+  
+        </div>
+      </section>
+      <form>
+        <h2>Add your score</h2>
+        <input type="text" placeholder="Your name" required id="name">
+        <input type="text" placeholder="Your score" required id="score">
+        <input type="submit" value="submit" id="submit">  
+      </form>
+    </main>
```

Comment:
- [ ] Following the project requirements, your HTML code should look like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png).

![](https://github.com/microverseinc/curriculum-javascript/raw/main/leaderboard/images/leaderboard_wireframe.png
)